### DISCUSSION TITLE
Issue while installing CaPTK on MacOS

### DISCUSSION THREAD

**Discussion by meghbhalerao** [CBICA/CaPTk#1396] [Q&A] [answered]
Hi all,
When I download and install CapTK on MacOS from here: https://www.nitrc.org/frs/downloadlink.php/11780 - I download the .pkg file from here and install, the install goes about smoothly, but after I install it, I am not able to locate where it is installed. Any help would be greatly appreciated.
Thanks,
Megh

macOS Big Sur 11.2

**Comment by sarthakpati** [comment, marked as answer]
It should be under `Applications`. Unfortunately, I am not using macOS, so cannot say for certainty (perhaps @AlexanderGetka-cbica could be of help?). If you are interested in the CLI applications, perhaps this would be useful:

https://cbica.github.io/CaPTk/FAQ.html#gs_FAQ_3
